pygount/common.py

```diff
@@ -10,7 +10,7 @@
 import warnings
 from typing import Generator, List, Optional, Pattern, Sequence, Union
 
-__version__ = "1.2.4"
+__version__ = "1.3.4"
```

Comment:
This should be 1.3.0.

Reply:
If you merge the current master, `__version__` is now managed by poetry. You will probably have a merge conflict here that can be resolved by accepting the changes on master and discarding your variant of this line.

Reply:
Oh. I love that you switched to poetry. I'll merge everything nicely within the next 1-2 days.

Reply:
Resolved with move to poetry.